Source/Events.Store.MongoDB/Processing/Streams/AbstractStreamProcessorState.cs

```diff
@@ -9,9 +9,13 @@
 namespace Dolittle.Runtime.Events.Store.MongoDB.Processing.Streams
 {
     /// <summary>
-    /// Represents the base state of an <see cref="AbstractScopedStreamProcessor" />.
+    /// Represents the base class for <see cref="StreamProcessorState"/> and <see cref="Partitioned.PartitionedStreamProcessorState"/>.
```

Comment:
Could we perhaps say in stead that this is the persisted version of `IStreamProcessorState` ?

Reply:
How about
```suggestion
    /// Represents the base class for <see cref="StreamProcessorState"/> and <see cref="Partitioned.PartitionedStreamProcessorState"/>,
    /// which represent the persisted versions of <see cref="IStreamProcessorState"/>.
```

I kinda want to have a comment about the derived classes in there too.

Reply:
Actually I like just the "persisted version of IStreamProcessorState" more.